README.md

```diff
@@ -174,9 +174,8 @@ Additional containers that are not directly integrated with the PostgreSQL Opera
 
 - [pgPool II](https://access.crunchydata.com/documentation/crunchy-postgres-containers/latest/container-specifications/crunchy-pgpool/)
 - [pg_upgrade](https://access.crunchydata.com/documentation/crunchy-postgres-containers/latest/container-specifications/crunchy-upgrade/)
-- [pgBench](https://access.crunchydata.com/documentation/crunchy-postgres-containers/latest/container-specifications/crunchy-pgbench/)
```

Comment:
We don't use this container anymore, do we?

Reply:
Honestly, we can remove this "Additional containers" section altogether.  It really doesn't make sense for `pg_upgrade` (which now _is_ integrated w/PGO), and we no longer build `pgBench` or `pgPool` containers for v5.

Reply:
May I ask why you decided to stop build `pgPool` containers?

Is there another solution to how I can split writes and reads? (proxy writes to master and reads to replicas)